todo/tests.py

```diff
@@ -0,0 +1,3 @@
+from django.test import TestCase
+
+# Create your tests here.
```

Comment:
Cover the code with tests so you can catch more bugs and when you change the code in the future it will help you avoid more bugs https://docs.djangoproject.com/en/4.1/topics/testing/